### DISCUSSION TITLE
V4.7 - PNP Search Results Web Part - RefinableString00 not displaying in the selectable properties list

### DISCUSSION THREAD

**Discussion by cosburback** [microsoft-search/pnp-modern-search#2576] [Q&A]
**Version used**
4.7 (this was also an issue in version 4.4.1)

**Describe the bug**
I have used RefinableString00 for column mapping. It never displays in the list of selectable properties. Is this a bug?

**To Reproduce**
1. Add a PnP Search Results web part
2. Click on selected properties
3. Scroll down and see that RefinableString00 is not listed.

**Expected behavior**
RefinableString00 should be listed just like the other Refinables that are mapped.

**Screenshots**
![image](https://user-images.githubusercontent.com/118854590/203421856-18865a90-f07c-498b-969b-0f228e0f3f87.png)

**Desktop (please complete the following information):**
 - Browser : Chrome
 - SharePoint Online

**Additional context**
I read that anything related to managed properties was not to be included here, but it just feels like this is a bug. If I'm wrong I apologise, and am happy to be corrected.

**Comment by wobba** [comment]
This happes at times and thinking to hardcode the list instead of using the unreliable non-api we use today. Resolution is to put the cursor in the selected properties box (as it's a combo box), manually enter the prop name and hit ENTER.
